src/operator/concat.cc

```diff
@@ -46,29 +46,39 @@ Operator* ConcatProp::CreateOperatorEx(Context ctx, std::vector<TShape> *in_shap
 DMLC_REGISTER_PARAMETER(ConcatParam);
 
 MXNET_REGISTER_OP_PROPERTY(Concat, ConcatProp)
-.describe(R"code(Concate a list of array along a given axis.
+.describe(R"code(Join input arrays along the given axis.
 
-The dimension sizes of the input arrays on the given axis should be the same.
+.. note:: `Concat` is deprecated. Use `concat` instead.
 
-For example::
+The dimensions of the input arrays should be the same except the axis along
+ which they will concatenated.
+The dimension of the output array along the concatenated axis will be equal
+to the sum of the corresponding dimensions of the input arrays.
 
-  x = [[1,1],[1,1]]
-  y = [[2,2],[2,2]]
-  z = [[3,3],[3,3],[3,3]]
+Example::
 
-  Concat(x,y,z,dim=0) = [[ 1.,  1.],
-                         [ 1.,  1.],
-                         [ 2.,  2.],
-                         [ 2.,  2.],
-                         [ 3.,  3.],
-                         [ 3.,  3.],
-                         [ 3.,  3.]]
+   x = [[1,1],[2,2]]
+   y = [[3,3],[4,4],[5,5]]
+   z = [[6,6], [7,7],[8,8]]
```

Comment:
basically this is pseudo code, not real code. If you paste this into python it won't work.